packages/apollo-server-express/src/ApolloServer.ts

```diff
@@ -177,6 +177,13 @@ export class ApolloServer extends ApolloServerBase {
       app.use(path, uploadsMiddleware);
     }
 
+    // mount graphql-playground-react on localhost so that it can be used offline
+    // TODO: how to ensure path to node_modules will always resolve, e.g. when used as a dependency or npm link?
+    app.use(
```

Comment:
hoping this can be replaced by something along the lines of

```js
app.use(
  '/graphql-playground-react',
  express.static(require.resolve('graphql-playground-react')),
);
```